Plate_2_data/2.cellprofiler_ic_processing/loaddata_edit_and_ic.ipynb

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">In this notebook, we create a LoadData CSV that contains the raw data (like in the previous module) without all rows except for rows correspopnding to the last z-plane as the name of the maximum projected images. These are the same and correct paths from the raw images to the maximum projected images. 

Then, we run the CellProfiler IC pipeline to calculate the illumination correction functions for all images per channel.
</pre><p> </p><p>Feel free to reject this suggestion if it fails to convey your message </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I was able to make this statement in the notebook more simple based on this suggestion.